pipeline/transform.py

```diff
@@ -0,0 +1,17 @@
+"""Transforms the plant data to make it ready for loading"""
+import pandas as pd
+
+
+def clean_plant_data(plant_df: pd.DataFrame) -> pd.DataFrame:
+    """Setting the types for dataframe columns and removing NaN values"""
+
+    plant_df['recording_at'] = pd.to_datetime(plant_df['recording_at'])
+    plant_df['last_watered'] = pd.to_datetime(plant_df['last_watered'])
+    plant_df['soil_moisture'] = pd.to_numeric(plant_df['soil_moisture'])
```

Comment:
Could you add the errors keyword arg in the function to make the values a NAN if not valid
```suggestion
    plant_df['soil_moisture'] = pd.to_numeric(plant_df['soil_moisture'], errors='COERCE')
```